code-samples/messaging/quick-start.js

```diff
@@ -1,22 +1,28 @@
+/*You get the environment parameters from your 
```

Comment:
This is a stylistic thing, but let's have comments in the following format:

```
/* Foo
   Bar
   Baz. */
```

Reply:
Please note the spaces separating the comment character and the comment. Also note the indentation of comments so everything is aligned along the same axis.